src/epub/templates/content.opf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:dc="http://purl.org/dc/elements/1.1/" unique-identifier="db-id"
+         version="3.0">
+
+    <metadata xmlns:opf="http://www.idpf.org/2007/opf">
+        <dc:title id="t1">Śpiewnik 21WDW Wersja: </dc:title>
+        <dc:creator id="editor">21. Warszawska Drużyna Wędrownicza</dc:creator>
+        <dc:identifier id="db-id">wdw21-experimental-3rmds92</dc:identifier>
+        <meta refines="#editor" property="role" scheme="marc:relators" id="role">edt</meta>
+        <meta property="dcterms:modified">2022-08-21T12:00:00Z</meta>
```

Comment:
Ideally this one should be also updated when you update the version.

Reply:
Done